doc/nrf/shortcuts.txt

```diff
@@ -200,6 +200,9 @@
 
 .. |trusted_execution| replace:: nRF5340 and nRF9160
 
+.. |samples_tfm_info| replace:: Starting from the |NCS| v2.0.0, TF-M is enabled by default for applications and samples that support hardware-enforced security by separation into the SPE and the NSPE.
```

Comment:
The way that I read that is t that by default you build with TF-M but in fact you want to say that when you enable hard-enforced security by separation then you build with TF-M no?